lib/file-info-view.coffee

```diff
@@ -10,10 +12,29 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?() or 'untitled')
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @addEventListener('click', clickHandler)
```

Comment:
What do you think about adding the listener on the `@currentPath` itself? In my (simpler) version I went with that idea because we are copying only the path so if someone extends the class with some other elements we won't affect them. With the listener on the `FileInfoView` itself clicking on the `@bufferModified` or any other (not yet existing) element will activate the click handler too even though only the `@currentPath` behaves as a link.